src/hex_http_httpc.erl

```diff
@@ -1,15 +1,16 @@
 -module(hex_http_httpc).
 -behaviour(hex_http).
--export([request/4]).
+-export([request/5]).
 
 %%====================================================================
 %% API functions
 %%====================================================================
 
-request(Method, URI, ReqHeaders, Body) ->
+request(Method, URI, ReqHeaders, Body, Options) ->
```

Comment:
nitpick: could you rename Options to AdapterConfig here and elsewhere?

Reply:
Yea, will do.